src/coord/src/coord.rs

```diff
@@ -280,41 +326,35 @@ impl Coordinator {
         self.worker_txs.len()
     }
 
-    /// Assign a timestamp for a read.
-    fn get_read_ts(&mut self) -> Timestamp {
-        let ts = self.get_ts();
-        self.last_op_was_read = true;
-        self.read_lower_bound = ts;
-        ts
+    fn system_timeline(&mut self) -> &mut Timeline {
+        self.timelines
+            .entry(TimelineId::EpochMilliseconds)
+            .or_insert_with(|| Timeline::new(0))
     }
 
-    /// Assign a timestamp for a write. Writes following reads must ensure that they are assigned a
-    /// strictly larger timestamp to ensure they are not visible to any real-time earlier reads.
-    fn get_write_ts(&mut self) -> Timestamp {
-        let ts = if self.last_op_was_read {
-            self.last_op_was_read = false;
-            cmp::max(self.get_ts(), self.read_lower_bound + 1)
-        } else {
-            self.get_ts()
-        };
-        self.read_lower_bound = cmp::max(ts, self.closed_up_to);
-        self.read_lower_bound
+    // Assign a timestamp for a table write.
+    fn get_table_write_ts(&mut self) -> Timestamp {
```

Comment:
I'm not sure about the intended design, but I think we'll want to have tables even for non-system timelines. Doesn't need to happen here, but the less this ossifies the better, imo.